frontera/contrib/backends/remote/codecs/json.py

```diff
@@ -112,10 +118,13 @@ def _request_from_object(self, obj):
 
     def decode(self, message):
         message = dict_to_bytes(super(Decoder, self).decode(message))
+        if message['type'] == 'links_extracted':
```

Comment:
Due to the conversion to bytes in the line above the values and keys will be bytes in message.